nservicebus/upgrades/encryption-key-identifiers.md

```diff
@@ -59,7 +60,7 @@ The following configuration examples demonstrate how to update existing endpoint
 
 #### XML Configuration
 
-It is possible to choose to keep the current encryption key and will still be able to decrypt messages already in flight:
+It is possible to keep the current encryption key and still be able to decrypt messages in flight:
 
 ```xml
```

Comment:
@Particular/docs-maintainers This file has inline code. Was not sure if it is worth to move this into snippets since the information in there seems to be fairly stable. Happy to do it if you think it is necessary